src/handlebar-helpers.js

```diff
@@ -0,0 +1,26 @@
+// list of useful handlebars helpers we can bundle
+// credit goes to https://stackoverflow.com/a/31632215 & https://stackoverflow.com/a/50427569
+//
+// TODO: maybe make this extensible via another CR field?
+
+const objectPath = require('object-path');
+
+const helpers = {
+  assign: function (varName, varValue, options) {
+    objectPath.set(options, `data.root.${varName}`, varValue);
```

Comment:
could result in undesirable behavior if varName is a number
```suggestion
    objectPath.set(options, 'data.root', objectPath.get(options, 'data.root', {}));
    objectPath.set(options, ['data', 'root', varName], varValue);
```

Reply:
good catch, fixed